Main.java

```diff
@@ -0,0 +1,15 @@
+package com.company;
+public class Main {
+    public static void main(String[] args)
+    {
+        for(int i=0;i<=5;i++){
```

Comment:
My bad for not writing the question properly, let's ask user for an input `n` and then use that to print the pattern, `n` would mean the number of stars user wants in the triangle's base.
```suggestion
        for(int i=0;i<=n;i++){
```